pages/generative-apis/how-to/query-vision-models.mdx

```diff
@@ -7,6 +7,7 @@ dates:
   posted: 2024-10-30
 ---
 import Requirements from '@macros/iam/requirements.mdx'
+import ChatCompVsResponsesApi from '@macros/ai/chat-comp-vs-responses-api.mdx'
```

Comment:
We might want to remove the mention of the Responses API for the vision models: the mistral models are currently incompatible with it and they represent the big majority of our vision models). We will reintroduce the notion once we fix the compatibility.